docs/variants/clevo_nv41/test-matrix.md

```diff
@@ -0,0 +1,128 @@
+# Test matrix - MSI Z690-A WIFI DDR4
+
+## About
+
+The test matrix is used to determine the scope of tests which the DUT is
+subjected from before the release of the new binary.
+
+## Test matrix - Clevo NV41MZ
```

Comment:
First line should be deleted, then move line with Clevo NV41MZ on top and remove `#` in some headers.

Reply:
improved